src/main/java/com/github/junrar/rarfile/FileHeader.java

```diff
@@ -67,12 +73,20 @@
 
     private final byte[] salt = new byte[SALT_SIZE];
 
+    private FileTime lastModifiedTime;
+
     private Date mTime;
```

Comment:
@gotson given the `FileTime` fields above are available and contain the most accurate precision which reflects the extended time format better (and it also makes this more like `ZipEntry`), I'm thinking it we can get rid of the `java.util.Date` fields and just get them from the `FileTime` equivalents when calling the getters (documenting the change accordingly, of course).

Ex.:

```java
public Date getMTime() {
    return toDate(lastModifiedTime);
}
```